db/migrate/20220705211136_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :author_id, null: false
```

Comment:
- [ ] this part of the migration should look something like the below:
```
t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
Thank you for the feedback, ill implement it